InfiniteLoopLMSSQL/src/com/st/lmssql/utils/ConnectionFactory.java

```diff
@@ -0,0 +1,42 @@
+package com.st.lmssql.utils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionFactory {
+
+	static Properties prop = new Properties();
+	static InputStream input = null;
+	public static String dbURL;
+	public static String user;
+	public static String pass;
+	
+	public static Connection getMyConnection() {
+		try {
+			input = new FileInputStream("resources/db.properties");

```

Comment:
This stream is leaked. Either explicitly close it in a `finally` block, or use try-with-resources.